### ISSUE TITLE
Linebreaks become quotes.

### ISSUE THREAD

**Issue by danhanly** [Elephant418/Markdownify#19]
I have a strange situation whereby line breaks are converted to quotes. 

Here's my input:

```
Hi

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam nec nisi vitae erat consectetur posuere et et lectus. Nullam vestibulum quis ligula quis interdum. Duis rutrum dignissim neque. Nullam mattis massa elit, at molestie purus feugiat a. Vivamus id leo leo. Sed euismod nec nisi eget elementum. 
```

Here's the output:

```
Hi  
> >   
> > Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam nec nisi vitae erat consectetur posuere et et lectus. Nullam vestibulum quis ligula quis interdum. Duis rutrum dignissim neque. Nullam mattis massa elit, at molestie purus feugiat a. Vivamus id leo leo. Sed euismod nec nisi eget elementum.   
```

How can I configure this so it doesn't happen?

**Comment by tzi** [comment]
Hi!

I do not see any use case where linebreaks could be converted to quotes.
I made some tests with your data, and I do not see the quotes :cry: 

Could you share more HTML / PHP code, so I can help you?

Cheers,
Thomas.

**Comment by danhanly** [comment]
It's something to do with the indent. It (for some reason) adds `'  >'` to the indent character, before it reaches the `handleTag_br` - I don't have the code in front of me at the moment, but I'll dig out the case when I get a chance.

**Comment by danhanly** [comment]
@tzi Okay, I have a clearer example for you.

The following text (anonymised):

```
Hi, <br /><br />Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam nec nisi vitae erat consectetur posuere et et lectus. Nullam vestibulum quis ligula quis interdum. Duis rutrum dignissim neque. Nullam mattis massa elit, at molestie purus feugiat a. Vivamus id leo leo. Sed euismod nec nisi eget elementum<br /><br />I hope this goes some way to answer your question, I'm sorry I can't be more specific.<br /><br />Regards,<br />Mike
```

Becomes this:

```
Hi, 
> > 
> >   
> >   
> > Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam nec nisi vitae erat consectetur posuere et et lectus. Nullam vestibulum quis ligula quis interdum. Duis rutrum dignissim neque. Nullam mattis massa elit, at molestie purus feugiat a. Vivamus id leo leo. Sed euismod nec nisi eget elementum  
> >   
> > I hope this goes some way to answer your question, I'm sorry I can't be more specific.  
> >   
> > Regards,  
> > Mike
```

Here's the code: 
``` 
// function __construct()

$this->markdown = new Converter();
```

```
// function formatText($html)

return $this->markdown->parseString($html);
```

**Comment by tzi** [comment]
Thanks for the details!

I'm still not reproducing this bug :cry: 
Could you tell which version of Markdownify you are using?

Cheers, 
Thomas.

**Comment by danhanly** [comment]
2.1.11

I'm going to try and reproduce this in a new skeleton project. I'll let you know what happens.

**Comment by tzi** [comment]
Thanks a lot!

I'm wainting for your feedbacks :wink:

**Comment by danhanly** [comment]
@tzi hopefully the PR I created here #20 will fix the issue and let you know what the cause of it was.

It's a design consideration mainly, but my PR adds flexibility on how you might want to use this package.

**Comment by tzi** [comment]
Fixed by d248586
